spec/unit/generator/generator_tags_spec.rb

```diff
@@ -64,7 +64,7 @@ def tags_mash_from_strings(tags_strings)
     context "with excluded and between tags" do
       let(:generator) { GitHubChangelogGenerator::Generator.new(between_tags: %w(1 2 3), exclude_tags: %w(2)) }
       it { is_expected.to be_a Array }
-      it { is_expected.to match_array(tags_mash_from_strings(%w(1 3))) }
+      it { is_expected.to match_array(tags_mash_from_strings(%w(1 2 3))) }
```

Comment:
Changed responsibilities often mean changed tests, removed tests and most often changed test descriptions. 

If behavior changes, tests change with it. 

Before we drop or change a test, we want to know that we check that behavior where it happens (now). 

So, where do we want to read about the excluding tags bit? What behavior supports that behavior?